v19.1/recommended-production-settings.md

```diff
@@ -43,25 +43,35 @@ For added context about CockroachDB's fault tolerance and automated repair capab
 
 ## Hardware
 
+### Terminology
```

Comment:
@bdarnell - can you review this new section?

Reply:
I don't know that we need to define CPU or Core here, because we don't really use these terms. We mainly just need to say that when we talk about CPU resources we're talking about vCPUs, which are also known as hyperthreads.

Reply:
We mention cores later but if you think this is overkill, I can just include this info as a note.

Reply:
Where do we mention cores? The only place I see is in reference to "shared-core" machines which link to cloud-specific docs, which seems fine. We want to avoid using the word "core" anywhere (except to say that in most cases there are two vCPUs per core).

Reply:
Just the mention of "shared-core", so will move this to a note instead.